hot-reload.md

```diff
@@ -44,7 +45,9 @@ app continues to execute from where it was prior to running the hot reload
 command. The code is updated and execution continues.
 
 A code change has a visible effect only if the modified Dart code is run again
-after the change. The next sections describe common situations where the
+after the change. Specifically, a hot reload causes all the existing widgets to rebuild. Code that is not invoked during the rebuilding of the widgets are not automatically re-executed.
```

Comment:
Last statement is not clear. Do you mean to say that `only code that is involved in rebuilding of the current widgets is automatically re-executed`?

Reply:
Thanks. Done